### ISSUE TITLE
Randomly moving agent (R3)

### ISSUE THREAD

**Issue by esfandia** [NMAI-lab/SAVI#2]

**Comment by ntartania** [comment]
So I've implemented a third agent, green, moving randomly and randomly dropping garbage in its path. It's a bit 'quick and dirty' because the randomness is delegated to the environment, and the two actions I added are just literals and it's hard-coded which agent does what.

**Comment by ntartania** [comment]
Agreed. At the moment nothing is implemented as internal actions -- the agents use the default agent class. I think the next step -- avoiding collisions -- is the one that will require an agent-specific class. It would be a lot cleaner. 
Also theoretically when several plans (intentions) are applicable, the selection is supposed to be non-deterministic. But I think in the implementation it just goes for the first one. (I discovered that when I tried implementing random movement by making different intentions all possible -- go right, go left, etc. -- it always did the first one).
